android/app/src/main/java/org/fossasia/openevent/adapters/DayScheduleAdapter.java

```diff
@@ -107,6 +109,17 @@ public void onBindViewHolder(DayScheduleViewHolder holder, int position) {
         }
         holder.slotLocation.setText(currentSession.getMicrolocation().getName());
 
+        Track currentSessionTrack = currentSession.getTrack();
+        holder.slotTrack.setText(currentSessionTrack.getName());
```

Comment:
![125f5dea-ff6d-451f-94c9-25116515dde1](https://cloud.githubusercontent.com/assets/19890557/26447046/568e99ac-4164-11e7-8de5-34681d01707b.jpg)
Check this out now.